tools/policy_sync/policy_sync.pt

```diff
@@ -171,7 +171,7 @@ define upload_template($data,$project_id,$param_alert_options,$param_governance_
         $response_array << $update_response
         $policy_template_href = $update_response["body"]["href"]
       end
-      if $param_publish == 1
+      if ( $param_publish == 1 ) && ( $item["name"] != "Policy Template Synchronization" )
```

Comment:
published_templates are scoped to the org. Is `$project_id` really holding the org ID? or just the project ID?